app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const db = require('./db/connection');
+
+db.connect()
+.then(() =>
+{
+    console.log("Database connected...");
+})
+.catch((err) =>
+{
+    console.log(err);
+})
+
+app.use('/api/topics', require('./routes/topics'));
+
+const PORT = process.env.PORT || 9090;
```

Comment:
Same with this and the app.listen below. They can both be removed. They are both built-in functionality that we get for free from express.